src/controller/data_model/controller-clusters.matter

```diff
@@ -2824,6 +2824,71 @@ client cluster IcdManagement = 70 {
   command access(invoke: manage) StayActiveRequest(): DefaultSuccess = 3;
 }
 
+/** This cluster supports remotely monitoring and, where supported, changing the operational state of an Oven. */
+client cluster OvenCavityOperationalState = 72 {
+  enum ErrorStateEnum : enum8 {
+    kNoError = 0;
+    kUnableToStartOrResume = 1;
+    kUnableToCompleteOperation = 2;
+    kCommandInvalidInState = 3;
+  }
+
+  enum OperationalStateEnum : enum8 {
+    kRunning = 1;
+    kPreheating = 64;
+    kPreheated = 65;
+    kCoolingDown = 66;
+    kSelfCleaning = 67;
+  }
+
+  struct ErrorStateStruct {
+    enum8 errorStateID = 0;
+    optional char_string<64> errorStateLabel = 1;
+    optional char_string<64> errorStateDetails = 2;
+  }
+
+  struct OperationalStateStruct {
+    enum8 operationalStateID = 0;
+    optional char_string<64> operationalStateLabel = 1;
+  }
+
+  critical event OperationalError = 0 {
+    ErrorStateStruct errorState = 0;
+  }
+
+  info event OperationCompletion = 1 {
+    enum8 completionErrorCode = 0;
+    optional nullable elapsed_s totalOperationalTime = 1;
+    optional nullable elapsed_s pausedTime = 2;
+  }
+
+  readonly attribute nullable char_string phaseList[] = 0;
+  readonly attribute nullable int8u currentPhase = 1;
+  readonly attribute optional nullable elapsed_s countdownTime = 2;
+  readonly attribute OperationalStateStruct operationalStateList[] = 3;
+  readonly attribute enum8 operationalState = 4;
```

Comment:
This seems to be a typed enum in the spec: OperationalStateEnum.

Is there a reason we kept this as enum8 instead of using that enum?